_dev/package.json

```diff
@@ -9,12 +9,14 @@
   "scripts": {
     "systemjs": "^0.19.31",
     "startServer": "node ../../../start.js",
-    "watch": "watchify sources/addin.ts -p tsify --debug -o ../addin.js -dv",
-    "build": "browserify sources/addin.ts -p tsify > ../addin.js",
-    "start": "start npm run startServer & start npm run watch"
+    "watch": "watchify sources/addin.ts -p tsify --debug -o ../public/addin.js -dv",
+    "build": "browserify sources/addin.ts -p tsify > ../public/addin.js",
+    "start": "start npm run startServer & start npm run watch",
+    "copyFiles": "copyfiles -u 2 ../public/addin.* ../public/registrationConfig.html ../../../MyGeotabDevTool/addins/DatabaseExportConfig/"
```

Comment:
Don't see the necessity of this script. As for me it's easier just put your repository into MyGeotabDevTool/addins/ folder and just run command 'npm run start'. This will start server, run watcher and open browser.

Reply:
I struggled with testing but happy for you not to accept this update. no problem

Reply:
how is this handled do I roll back these changes?

Reply:
You can roll back these changes and commit

Reply:
I would like to keep this. It should not impact you. It allows me to debug in my world. I have not succeeded in debugging using your strategy. Ok for you?

Reply:
Hi Nick, I have never succeeded in doing the first thing you stated. Do I place the whole project in the addins folder or merge the project into the MyGeotabDevTool environment. So I would only have 1 add-in in the DevTool for each project, is this correct? I would really like to understand how to achieve this. Can we have a call to discuss? thanks

Reply:
Just place your project in the "addins" folder. The path to html file should looks like: addins -> "your add-in folder" -> "your add-in index file".html. Then move to _dev folder and run start script (use command "npm run" or "npm run start")

Reply:
But this scheme won't work with "public" subfolder. I think the better option will be moving html and js back to the root folder. If you want to have a "clean" version you can update your copyFiles script to put necessary files to "public" folder.

Reply:
as agreed I will leave this in place for the moment. No global impact on others.